app/src/main/java/com/android/unio/model/event/Event.kt

```diff
@@ -37,3 +46,15 @@ data class Event(
 ) {
   companion object
 }
+
+enum class EventType(val color: Color, val text: String) {
+  FESTIVAL(eventTypeFestival, "festival"),
+  APERITIF(eventTypeAperitif, "aperitif"),
+  NIGHT_PARTY(eventTypeNightParty, "night party"),
+  JAM(eventTypeJam, "jam"),
+  NETWORKING(eventTypeNetworking, "networking"),
+  SPORT(eventTypeSport, "sport"),
+  TRIP(eventTypeTrip, "trip"),
+  OTHER(eventTypeOther, "other")
```

Comment:
I am not sure to understand but I don't think so ! 
I think in order to preserve a color identification of the type of Event (so that we can understand which event is a festival in the blink of an eye :), it should be the same when going to light/dark mode